Ginger/GingerCoreNET/ValueExpressionLib/ValueExpression.cs

```diff
@@ -1365,12 +1367,17 @@ private void ReplaceGeneralFunctionsWithValue(string p, string[] a)
                         FuncSplit.RemoveAll(x => x.Equals(""));
                         List<string> parameters = Regex.Split(FuncSplit[0].ToString(), paramPattern).ToList<string>();
                         parameters.RemoveAll(y => y.Equals(""));
-                        parameters = parameters.Where(z => z.Contains("\"")).Select(z => z.Replace("\"", "")).ToList<string>();
-
+                        
```

Comment:
why to have those changes?

Reply:
This is because earlier it was removing all the "(quotes) from the actual string as well, instead of that we can just trim it to remove only from the start and end of the string.